cmd/operator/main.go

```diff
@@ -68,10 +75,13 @@ func main() {
 		publicNamespace = flag.String("public-namespace", operator.DefaultPublicNamespace,
 			"Namespace in which the operator reads user-provided resources.")
 
-		tlsCert     = flag.String("tls-cert-base64", "", "The base64-encoded TLS certificate.")
-		tlsKey      = flag.String("tls-key-base64", "", "The base64-encoded TLS key.")
-		caCert      = flag.String("ca-cert-base64", "", "The base64-encoded certificate authority.")
-		webhookAddr = flag.String("webhook-addr", ":10250",
+		tlsCert      = flag.String("tls-cert-base64", "", "The base64-encoded TLS certificate.")
+		tlsKey       = flag.String("tls-key-base64", "", "The base64-encoded TLS key.")
+		caCert       = flag.String("ca-cert-base64", "", "The base64-encoded certificate authority.")
+		certFile     = flag.String("cert-file", defaultTLSDir+"tls.crt", "Path to TLS certificate for webhook server.")
```

Comment:
What if we have non-webhook server TLS certificates in the future. Maybe we should be more detailed here with the flag name.